docs/pages/enterprise/workflow/ssh-approval-pagerduty.mdx

```diff
@@ -79,6 +79,15 @@ Run `./install` from `teleport-pagerduty` or place the executable in the appropr
 </TabItem>
 </Tabs>
 
+<Details title="Installing via Helm?">
```

Comment:
From what I can tell this will be a drop down box underneath the set of tabs that have various options for deploying. Is there a reason `helm` isn't included as another item in the list of tabs and instead as a separate area?